subsys/bluetooth/host/Kconfig

```diff
@@ -604,11 +604,11 @@ config BT_ECC
 	  This option adds support for ECDH HCI commands.
 
 config BT_TINYCRYPT_ECC
-	bool "Use TinyCrypt library for ECDH"
```

Comment:
I have no strong opinion on this line. But I'm fine with keeping the prompt as is

Reply:
No strong opinion either, but thought it valuable to explain that it is host-side when the feature can be enabled in both host or controller.